storybook/stories/_support/StoryProvider.js

```diff
@@ -88,6 +93,38 @@ export const WALLETS = [
   },
 ];
 
+export const WALLETS_V2 = [
```

Comment:
@daniloprates maybe would be better to name this like `SHELLEY_WALLETS`. What do you think about that?

Reply:
I would keep it WALLETS_V2 for now. We don't use shelley anywhere...
cc @daniloprates @tomothespian

Reply:
@nikolaglumac Deal, approving